src/App.tsx

```diff
@@ -2,108 +2,33 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList/PostList';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
+
+function getUserById(userId: number): User | null {
```

Comment:
will be better to name this function `findUserById`, because you can't be sure that you'll get user, you can return null from this function